### PR TITLE
Add Chromium versions for api.FetchEvent.request

### PR DESCRIPTION
**Pull request by queengooborg**
This PR adds real values for Chromium (Chrome, Opera, Samsung Internet, WebView Android) for the `request` member of the `FetchEvent` API, based upon commit history and date.

Commit: https://source.chromium.org/chromium/chromium/src/+/451a831fc3cb3c0b845298abbdde172219444d9f

### CODE REVIEW

**Comment by queengooborg** [comment]
The date of this particular commit is months before Chrome 40's feature freeze, which is when this event interface was implemented.  As such, I also set this feature to Chrome 40.

**Comment by foolip** [comment]
I thought this would be easy to verify, but I guess the reason you didn't use the collector here is that it's broken. I've sent https://github.com/foolip/mdn-bcd-collector/pull/1479 + https://github.com/foolip/mdn-bcd-collector/pull/1481 but that won't be enough to test this I think.

https://developer.mozilla.org/en-US/docs/Web/API/Request#browser_compatibility says that `Request` was in Chrome 42, and the same is true of [`fetch()`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API#browser_compatibility) itself, so I guess the `FetchEvent` data is just wrong. Can you bump it to 42? That could still be wrong, but at least less wrong than now.

File: api/FetchEvent.json [hunk 1]
```diff
@@ -356,10 +356,10 @@
           "spec_url": "https://w3c.github.io/ServiceWorker/#fetch-event-request",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "40"
```
Comment (foolip) [inline api/FetchEvent.json:359]: This change, but everywhere and mirrored:

```suggestion
              "version_added": "42"
```

**Review by foolip — CHANGES_REQUESTED** [review]

**Comment by foolip** [comment]
Trying to confirm this again, I found an interesting note that might explain this. https://caniuse.com/fetch says "Only available in Chrome and Opera within ServiceWorkers" for Chrome 40-41, and if that's true for the whole fetch API things would make more sense. That would mean that `FetchEvent` actually was supported from Chrome 40, and we should mark `fetch()`, `Request` and probably a bunch of other things as supported from Chrome 40.

@queengooborg do you want to tackle that in this PR, or perhaps revert back to 40 and file an issue about the mess?

**Comment by queengooborg** [comment]
I've reverted the changes from Chrome 42 -> 40 and filed an issue.  We should be able to fix this in the next collector version!

**Review by foolip — APPROVED** [review]

**Pull request merged**